src/components/RecipesDisplay/RecipeBrief.css

```diff
@@ -0,0 +1,82 @@
+.recipe-brief-container{
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  margin: 1.25em;  /* 20px */
+
+  font-family: Poppins;
+}
+
+.recipe-card{
+  width: 300px;
+  height: 225px;
+  margin-bottom: 1.94em; /* 31px */
```

Comment:
Ideally, your units should be in multiples of 0.25em or multiples of 4px

Reply:
update px to multiples of 4.